samples/xbee/demos/EndToEndSample/main.py

```diff
@@ -98,5 +118,30 @@ def drm_request_callback(target, request):
             device.close()
 
 
+def is_old_firmware():
+    """
+    Returns whether the firmware running is an old version or not in
+    terms of data point upload capabilities.
+
+    Returns:
+        ``True`` if the device firmware is old, ``False`` otherwise.
```

Comment:
NP: add the return type.

```python
    """
    Returns whether the firmware running is an old version or not in
    terms of data point upload capabilities.

    Returns:
        Boolean: `True` if the device firmware is old, `False` otherwise.
    """
```

Reply:
Thanks, done.